catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -45,13 +47,22 @@
      * @param values the values accepted by this validator
      * @throws IllegalArgumentException if {@code values} is null or empty
      */
-    public EnumerationValidator(final Set<String> values) {
+    public EnumerationValidator(boolean ignoreCase, final Set<String> values) {
         Preconditions.checkArgument(CollectionUtils.isNotEmpty(values),
                 "Must specify at least one possible enumeration value.");
 
-        this.values = values.stream()
-                .filter(Objects::nonNull)
-                .collect(Collectors.toSet());
+        this.ignoreCase = ignoreCase;
+
+        if (!ignoreCase) {
```

Comment:
I'm thinking these blocks could be combined..
`Stream<String> stream = values.stream()
stream.filter(Objects::nonNull);
if(ignoreCase) {
stream..map(String::toLowerCase);
}
this.values = stream.collect(Collectors.toSet());
`